acceptance/cmd_runner.go

```diff
@@ -39,7 +39,7 @@ func (r *cmdRunner) RunCommand(env map[string]string, args ...string) (string, s
 }
 
 func (r *cmdRunner) RunStreamingCommand(out io.Writer, env map[string]string, args ...string) (string, string, int, error) {
-	exports := make([]string, len(env))
+	var exports []string
```

Comment:
What's wrong with `make`ing a slice if you know the length beforehand?

Reply:
Nothing wrong, but you'd want to index the slice to add items instead of `append` like below. `append` is going to grow the slice and then you have `len(env)` zero-values at the beginning of the slice.